usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/whiteboardZoneClinet.js

```diff
@@ -0,0 +1,83 @@
+//
```

Comment:
is this file necessary? i notice that the filename is misspelled and the top of the file here doesn't match the filename